WalletWasabi.Fluent/ViewModels/Dialogs/ShowQrCameraDialogViewModel.cs

```diff
@@ -19,10 +20,11 @@ public partial class ShowQrCameraDialogViewModel : DialogViewModelBase<string?>
 	[AutoNotify] private string _errorMessage = "";
 	[AutoNotify] private string _qrContent = "";
 
-	public ShowQrCameraDialogViewModel(Network network)
+	public ShowQrCameraDialogViewModel(UiContext context, Network network)
 	{
 		_qrReader = new(network);
```

Comment:
@YohDeadfall it looks to me as this is the kind of external service that we want to abstract away as a `UiContext` component.

In the way this ViewModel is currently written, it will be hard to write unit tests for it. And even if you manage to do it, you're not really testing the ViewModel in isolation, you're testing the `WebcamQrReader` class together with it.

Ideally I'd `extract interface` from the `WebcamQrReader` class and add that interface to the `UiContext`, just like we did with the QR Code Generator.

Reply:
That's in another PR addressing that. See #10479. I just don't want to put that thing there just because CI complains, it's unrelated to camera handling.